ci-operator/step-registry/openshift-extended/netobserv-test/openshift-extended-netobserv-test-ref.yaml

```diff
@@ -0,0 +1,58 @@
+ref:
+  as: openshift-extended-netobserv-test
```

Comment:
This step can be cleanup as it's not used now. If we need any env in this file, move it to *chain.yaml.

Reply:
done, thanks!